DNN Platform/DotNetNuke.Web/Api/Internal/ServicesRoutingManager.cs

```diff
@@ -258,12 +261,11 @@ private IEnumerable<IServiceRouteMapper> GetServiceRouteMappers()
                 IServiceRouteMapper routeMapper;
                 try
                 {
-                    routeMapper = Activator.CreateInstance(routeMapperType) as IServiceRouteMapper;
+                    routeMapper = ActivatorUtilities.CreateInstance(serviceProvider, routeMapperType) as IServiceRouteMapper;
```

Comment:
I like adding the ActivatorUtlities here it enables us to use either reflection or the container. I was wondering if it made sense to register all implementations of `IServiceRouteMapper` as a transient service. That way we can just depend on the container and it would never use reflection here.

Reply:
I had been thinking about that (especially with some of the trickier scenarios I looked at in #3933).  In this case, we're only instantiating these instances one time on startup, so it doesn't seem to me like it'd be worth it.  But definitely a route we can look at for other extension points that are used multiple times.

Reply:
That makes sense, the only scenario I can think of which would make sense would be route localization. If you want to inject a localization service to the `IServiceRouteMapper` which would handle generating localized routes.

The developer could still just register the mapper in their own implementation of `IDnnStartup`.